benchmark/nixlbench/README.md

```
@@ -257,9 +261,9 @@ cd ucx
 make -j$(nproc) && sudo make install
 ```
 
-#### etcd-cpp-api Installation (Required)
+#### etcd-cpp-api Installation (Optional for ETCD Runtime)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)